source/_integrations/webostv.markdown

```diff
@@ -239,3 +239,7 @@ automation:
         data:
           icon: "/home/homeassistant/images/doorbell.png"
 ```
+
+## Notes
+
+- If Home Assistant and your TV are not on the same subnet, you need to open Port 3000 with Protocol TCP from Home Assistant to your TV.
```

Comment:
I'm confused about how subnets are related to the port? Don't you mean the firewall or more explicit routing you might have placed between your subnets?

Reply:
```suggestion
- If Home Assistant and your TV are not on the same network, you need to create a Firewall Rule, which allows a Connection on Port 3000 with Protocol TCP from Home Assistant to your TV.
```
I tried to keep this as simple as possible, but I agree that the wording was not ideal.
Hopefully my suggestion makes it more clear.